Create 2019 Summit: Portland

rachel-kauff opened the pull request:
Create new page for 2019 Summit! (Working draft here for reference: https://docs.google.com/document/d/1P3eN2-dVnYilctZg6P8nm1-Urv2G-u7oN79jIrVsc5o/edit#)

rachel-kauff committed:
Create 2019 Summit: Portland

Create new page for 2019 Summit! (Working draft here for reference: https://docs.google.com/document/d/1P3eN2-dVnYilctZg6P8nm1-Urv2G-u7oN79jIrVsc5o/edit#)

akaroleff reviewed (changes requested):
line 93 should read "and cowork" instead of "an cowork" it also says "retreat"
line 113 uses a dash format instead of the word "on" that is different from line 112
line 138 says retreat instead of summit
line 151 says retreat instead of summit
line 159 says retreat instead of summit

lines 75 & 79 links will break once we update the link title to say summit instead of retreat

rachel-kauff committed:
Update 2019-Summit.md

FIXED- line 93 should read "and cowork" instead of "an cowork" it also says "retreat"
FIXED - line 113 uses a dash format instead of the word "on" that is different from line 112
FIXED - line 138 says retreat instead of summit
FIXED - line 151 says retreat instead of summit 
FIXED - line 159 says retreat instead of summit

NEED TO CHANGE LINK TITLES BEFORE CORRECTING- lines 75 & 79 links will break once we update the link title to say summit instead of retreat
